### DISCUSSION TITLE
Yi 1080p RFUS new Firmware

### DISCUSSION THREAD

**Discussion by sasciotto** [roleoroleo/yi-hack-Allwinner-v2#524] [Q&A]
Ciao, quando sarà possibile compatibilità con nuovo Firmware Version 11.1.35.3 per YI 1080P RFUSY62Q6L?
Grazie per la vostra risposta.

**Comment by roleoroleo** [comment]
Which suffix?

**Comment by Gregory-hard-bug** [comment]
Hello. It turned out to flash ya291ga, but after that, as soon as I try to work with the web page, the camera restarts. But without a flash card, everything works fine. I tried different cards and different chargers, nothing helped...

**Reply by roleoroleo** [reply]
Check the yi firmware version.
